dev/integration_tests/ui/lib/driver.dart

```diff
@@ -59,9 +59,9 @@ class DriverTestAppState extends State<DriverTestApp> {
                 DropdownButton<Letter>(
                   key: const ValueKey<String>('dropdown'),
                   value: _selectedValue,
-                  onChanged: (Letter newValue) {
+                  onChanged: (Letter? newValue) {
                     setState(() {
-                      _selectedValue = newValue;
+                      _selectedValue = newValue!;
```

Comment:
I think `_selectedValue` should be nullable, and then you won't need this `!`.

Reply:
Hello @gspencergoog 
```Letter _selectedValue = Letter.a;```
_selectedValue can not be nullable because it has default value

Reply:
Well, the default could be set in an `initState`. I suppose it doesn't matter though, because there aren't any null values in the list. OK, leave the `!`, I guess.